xmodule/library_content_block.py

```diff
@@ -264,6 +242,108 @@ def _publish_event(self, event_name, result, **kwargs):
         self.runtime.publish(self, f"edx.librarycontentblock.content.{event_name}", event_data)
         self._last_event_result_count = len(result)  # pylint: disable=attribute-defined-outside-init
 
+    def available_children(self) -> list[tuple[str, str]]:
```

Comment:
I exposed this as a public method so that unit tests can use it without duplicating the same logic.

The underlying implementation is still a static method so that make_selection can use it.